examples/driving_in_traffic/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM tensorflow/tensorflow:2.4.0-gpu
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install libraries
+RUN apt-get update --fix-missing && \
+    apt-get install -y \
+        software-properties-common && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install -y \
+        ffmpeg \
```

Comment:
Why is `ffmpeg` needed? Envision?

Reply:
`ffmpeg` is needed to create videos from rgb images, to be displayed in tensorboard, to visualize training/evaluation progress.